src/components/MultiHopTrade/components/MultiHopTradeConfirm/hooks/useAllowanceApproval.tsx

```diff
@@ -89,9 +89,12 @@ export const useAllowanceApproval = (
     stopPollingBuildApprovalTx,
   ])
 
-  return {
-    isLoading,
-    executeAllowanceApproval,
-    approvalNetworkFeeCryptoBaseUnit,
-  }
+  return useMemo(
```

Comment:
Don't return `useMemo()` directly and assign this expression to `value` before returning it

Reply:
solved as per
https://github.com/shapeshift/web/pull/6338#discussion_r1549017924
5e35960fcdd69296fff2972739c36bdf9af68c4c